include/crm/common/ipc_attrd_internal.h

```diff
@@ -146,6 +146,8 @@ int pcmk__attrd_api_refresh(pcmk_ipc_api_t *api, const char *node);
  * \param[in]     value      The attribute's new value, or NULL to unset
  * \param[in]     dampen     The new time to wait value, or NULL to unset
  * \param[in]     set        ID of attribute set to use (or NULL for first)
+ * \param[in]     block      Which block to update this attribute in?
+ *                           (XML_TAG_ATTR_SETS or XML_TAG_UTILIZATION)
```

Comment:
I'm thinking this could just be a flag in pcmk__node_attr_opts

Reply:
Yeah, that works as long as we won't want to set them into another block in the future.

Reply:
I don't expect any others, and it's internal API so at least easy to change

Reply:
Fixed.